### ISSUE TITLE
aarvy91 L02E12

### ISSUE THREAD

**Issue by ravisoni31** [mikeudacity/robo-pace-betatest#108]
There are many issues in this session.

(1) It throws errors if we use cv2.imshow() function. ( I was able to get correct answer by importing matplotlib.pyplot)

(2) Are we reading the correct image? I saw a bit different image than shown at the top of this lesson. ( I could also see that it is a ShutterStock image.) 

(3) It accepts the answer even with the incorrect code or no code!

**Comment by ravisoni31** [comment]
And

(4) It would be better to show some pictures/sample codes of OpenCV, image channels, etc before jumping into a code task.

**Comment by ravisoni31** [comment]
I just tried this again, and it prints "libdc1394 error: Failed to initialize libdc1394" along with the output images.

**Comment by cangelogamboa** [comment]
Yes, thanks for trying it again. I "fixed" the problem between your first post and your second one. Now I have to fix the error message. That will take a little longer. Thank you for letting me know.
